test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -105,26 +149,33 @@ cd "$top"
 (build_release mongodb-3.6)
 (build_release mongodb-4.0)
 (build_release mongodb-4.2)
-#(build_release mongodb-4.4)
+(build_release mongodb-4.4)
 (build_release "develop")
 
 # Run format in each release for supported access methods.
 (run_format mongodb-3.4 "fix row var")
 (run_format mongodb-3.6 "fix row var")
 (run_format mongodb-4.0 "fix row var")
 (run_format mongodb-4.2 "fix row var")
-#(run_format mongodb-4.4 "row")
-(run_format "develop" "row" "-B")
+(run_format mongodb-4.4 "row")
+(run_format "develop" "row")
 
 # Verify backward compatibility for supported access methods.
 (verify_release mongodb-3.6 mongodb-3.4 "fix row var")
 (verify_release mongodb-4.0 mongodb-3.6 "fix row var")
 (verify_release mongodb-4.2 mongodb-4.0 "fix row var")
-#(verify_release mongodb-4.4 mongodb-4.2 "fix row var")
-#(verify_release develop mongodb-4.4 "row")
-(verify_release develop mongodb-4.2 "fix row var")
+### (verify_release mongodb-4.4 mongodb-4.2 "fix row var")
+### (verify_release develop mongodb-4.4 "row")
+(verify_release develop mongodb-4.2 "row")
 
 # Verify forward compatibility for supported access methods.
+### (verify_release mongodb-4.2 mongodb-4.4 "row")
 (verify_release mongodb-4.2 develop "row")
+### (verify_release mongodb-4.4 develop "row")
+
+# Upgrade/downgrade testing for supported access methods.
+### (upgrade_downgrade mongodb-4.2 mongodb-4.4 "row")
+(upgrade_downgrade mongodb-4.2 develop "row")
+### (upgrade_downgrade mongodb-4.4 develop "row")
```

Comment:
Can we enable those commented-out test lines? `mongodb-4.4` should have a similar source as `develop` branch, in a slightly delayed fashion.

Reply:
No, `mongodb-4.4` doesn't currently work.